INV4/pallet-inv4/src/lib.rs

```diff
@@ -233,19 +248,28 @@ pub mod pallet {
     #[pallet::event]
     #[pallet::generate_deposit(pub(crate) fn deposit_event)]
     pub enum Event<T: Config> {
+        /// An IP Set was created
         Created(T::AccountId, T::IpId),
+        /// An IP Set was destroyed/deleted
         Destroyed(T::AccountId, T::IpId),
+        /// IpInfo (IPS) struct updated in storage to hold either new assets, new metadata, or both
         Appended(T::AccountId, T::IpId, Vec<u8>, Vec<AnyIdOf<T>>),
+        /// IpInfo (IPS) struct updated: assets removed from IPS. Optionally, new metadata set
         Removed(T::AccountId, T::IpId, Vec<u8>, Vec<AnyIdWithNewOwner<T>>),
+        /// Replicas of this IP Set are now allowed
         AllowedReplica(T::IpId),
+        /// Replicas of this IP Set are no longer allowed
         DisallowedReplica(T::IpId),
+        /// A replica of this IP Set was created
         ReplicaCreated(T::AccountId, T::IpId, T::IpId),
 
+        /// Sub tokens were minted/IPF was created???
         Minted(
             (T::IpId, Option<T::IpId>),
             T::AccountId,
             <T as pallet::Config>::Balance,
         ),
+        /// Sub tokens were burned/IPF was deleted???
```

Comment:
Tokens burned, you won't find IPF related events here, as those are defined in the IPF pallet.
```suggestion
        /// Sub tokens were burned
```